roles/ipaserver/library/ipaserver_prepare.py

```diff
@@ -355,6 +354,7 @@ def main():
     options.setup_ca = ansible_module.params.get('setup_ca')
     options.random_serial_numbers = ansible_module.params.get(
         'random_serial_numbers')
+    options._random_serial_numbers = options.random_serial_numbers
```

Comment:
ca.install_check is setting `options._random_serial_numbers`. See also comment "# These are set by ca.install_check and need to be passed to ca.install ..."

Reply:
I agree. PR was updated.